tests/core/methods/test_tdvp.py

```
@@ -554,3 +555,45 @@ def test_split_truncation_distribution_reconstructs_optimal_rank(distr: str) ->
+        (left_env, right_env, op),
+        ket.shape,
+    )
+
+    y1 = project_site(left_env, right_env, op, ket).reshape(-1)
+    y2 = H_eff @ ket.reshape(-1)
+
+    np.testing.assert_allclose(y1, y2, atol=1e-12)
+
+
+def test_dense_vs_project_bond() -> None:
+    l, r = 3, 4
```

Ruff (0.14.7)

Ambiguous variable name: `l`

(E741)